src/i18n/en-US/notifications.ts

```diff
@@ -103,12 +124,12 @@ const notifications = {
     },
     additionalConfigurations: {
       NEW_MODEL_PLAN: 'When a new Model Plan is created',
-      dateChanges: 'When model dates change',
+      datesChanged: 'When model dates change',
       whichModel: 'Which models?',
       whichModelTypes: {
-        all: 'All models',
-        following: 'Models I’m following',
-        mine: 'My Model Plans'
+        ALL_MODELS: 'All models',
+        FOLLOWED_MODELS: 'Models I’m following',
+        MY_MODELS: 'My Model Plans'
```

Comment:
Not sure why these are capitalized differently?  Is it being keyed on by a value somewhere?

Reply:
the `whichModelTypes` are in all caps to mimic the enums, as it will be sent along with the payload to the update mutation